src/test-helpers.ts

```diff
@@ -102,10 +102,14 @@ export async function getFakeStoreAdapter({
   readStub,
   readSinceStub,
   saveStub,
+  lastEventOf,
+  exists,
 }: {
   readStub?: any,
-  readSinceStub: any,
+  readSinceStub?: any,
   saveStub?: (evt: any) => Promise<undefined>,
+  lastEventOf?: (pattern: string) => Promise<Event<any, any>>,
+  exists?: (id: string) => Promise<boolean>,
```

Comment:
The actual PG definition of `exists` resolves to the first row of the result, not a `boolean`. Which one is correct?

Reply:
Resolved, it should return boolean as specified here.